clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -42,9 +42,9 @@
                                                        + "servers (you may want more than one, though, in case a server is down).";
 
     public static final String CLIENT_DNS_LOOKUP_CONFIG = "client.dns.lookup";
-    public static final String CLIENT_DNS_LOOKUP_DOC = "<p>Controls how the client uses DNS lookups.</p><p>If set to <code>use_all_dns_ips</code> then, when the lookup returns multiple IP addresses for a hostname,"
-            + " they will all be attempted to connect to before failing the connection. Applies to both bootstrap and advertised servers.</p>"
-            + "<p>If the value is <code>resolve_canonical_bootstrap_servers_only</code> each entry will be resolved and expanded into a list of canonical names.</p>";
+    public static final String CLIENT_DNS_LOOKUP_DOC = "Controls how the client uses DNS lookups. If set to <code>use_all_dns_ips</code> then, when the lookup returns multiple IP addresses for a hostname,"
```

Comment:
Why do we need to remove `<p>` in this case?

Reply:
These new paragraphs inserted unnecessary whitespace and it looked weird having paragraphs only containing very short sentences.